### PR TITLE
Ashley/my investments

### PR DESCRIPTION
**Pull request by ashleynguyen27**
*** Needs to be merged after cindy's prs ***
[//]: # "These comments meant for your reference, they are invisible and don't need to be deleted"

## What's new in this PR?
Updating investments page to hifi

### How to Review
My investments page should reflect what is on figma

### Related PRs

[//]: # "Optional - related PRs you're waiting on
/ PRs that will conflict, etc"
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

### Next steps

[//]: # "What doesn't work yet, what's NOT in this 
PR/has to be done "
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

### Screenshots

[
image
//]: # '#### YOUR SCREENSHOTS BELOW ########'
[//]: # '############## END ##################'

### Design Status
HIFI
[//]: # 'If this is a frontend PR, what is the expected 
status of the UI in this PR (lo, mid, high- fi?'
[//]: # '####### LOFI/MIDFI/HIFI ###########'
[//]: # '############## END ##################'

CC: @dfangshuo

### CODE REVIEW

File: src/screens/general/components/InvestmentsPieGraph.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            nameKey="x"
+            valueKey="y"
+            cx={200}
+            cy={150}
+          />
+        }
+      />
+    );
+  }
+}
```
Comment (cindyzhang12) [inline src/screens/general/components/InvestmentsPieGraph.js:50]: ooo nice labelComponent LOL

File: src/screens/general/components/utils/math.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+export const getXOffsetMultiplayerByAngle = angle =>
+  Math.cos(angle - 90 * radian);
+export const getYOffsetMultiplayerByAngle = angle =>
+  Math.sin(angle - 90 * radian);
+export const getXOffset = (offset, angle) =>
+  offset * getXOffsetMultiplayerByAngle(angle);
+export const getYOffset = (offset, angle) =>
+  offset * getYOffsetMultiplayerByAngle(angle);
+export const getAverage = array =>
+  array.reduce((acc, cur) => acc + cur, 0) / array.length;
```
Comment (cindyzhang12) [inline src/screens/general/components/utils/math.js:11]: wow many maths

Comment (dfangshuo) [reply src/screens/general/components/utils/math.js:11]: Rename: `chartUtils.js`

File: src/styles/Investments.css [hunk 3]
```diff
@@ -122,9 +148,9 @@
 .green-check {
```
Comment (dfangshuo) [inline src/styles/Investments.css:148]: prefix all css classes here with investment

File: src/screens/general/components/Label.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+const Label = props => {
+  const {
+    datum,
+    innerRadius,
+    radius,
+    slice: { startAngle, endAngle },
+    nameKey,
+    cx,
+    cy
```
Comment (dfangshuo) [inline src/screens/general/components/Label.js:14]: Is this the label for the chart? If so, please give it a more specific name

File: src/screens/general/components/Label.js [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  }
+
+  return (
+    <g>
+      {name.split('\n').map((sub, index) => (
+        <text
+          x={x}
+          y={y + index * 18}
+          textAnchor={textAnchor}
+          fill="#395578"
```
Comment (dfangshuo) [inline src/screens/general/components/Label.js:47]: declare this constant as a color at the top of the file

File: src/screens/general/components/LabelLine.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { getXOffset, getYOffset } from './utils/math';
+
+const LabelLine = props => {
+  const { cx, cy, midAngle, middleRadius, radius } = props;
+  const xStart = cx + getXOffset(middleRadius, midAngle);
+  const yStart = cy + getYOffset(middleRadius, midAngle);
```
Comment (dfangshuo) [inline src/screens/general/components/LabelLine.js:8]: Similarly - rename to make it more specific

**Pull request merged**